### PR TITLE
feat: script to copy across updated files from `main_repo`

### PR DESCRIPTION
**Pull request by samricotta**
The following issue tracks the issue to ensure docs are all up to date and aligning with the main sdk repo part of the revamp [epic ](https://github.com/cosmos/cosmos-sdk/issues/15705)

https://github.com/cosmos/cosmos-sdk/issues/17642

### CODE REVIEW

File: .github/workflows/copy-md.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Copy MD Files
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
```
Comment (tac0turtle) [inline .github/workflows/copy-md.yml:5]: can we make this twice a day and manual as well

File: sync_script.sh [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+# Store the current working directory in WORK_DIR
+WORK_DIR=$(pwd)
+
+# Remove any existing 'cosmos-sdk' directory and clone the remote repository
+rm -rf ./cosmos-sdk
+git clone "$REMOTE_REPO_URL" cosmos-sdk
+
+# Read the versions from a JSON file and remove the 'v' prefix
+VERSIONS=($(jq -r '.[]' versions.json))
```
Comment (tac0turtle) [inline sync_script.sh:14]: versions folder needs to be updated to include 0.50

**Review by tac0turtle — APPROVED** [review]
uACK

File: .github/workflows/copy-md.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+name: Copy MD Files
+
+on:
+  workflow_dispatch:
+    schedule:
+      - cron: '0 0,12 * * *'
```
Comment (tac0turtle) [inline .github/workflows/copy-md.yml:6]:
```suggestion
    schedule:
      - cron: '0 0,12 * * *'
    workflow_dispatch
```

for manual pushes

File: .github/workflows/copy-md.yml [hunk 4]
```diff
@@ -1,9 +1,9 @@
 name: Copy MD Files
 
 on:
-  schedule:
-    - cron: '0 0 * * *'
-
+  workflow_dispatch:
```
[the remark this answers is not in the document]
Comment (tac0turtle) [reply .github/workflows/copy-md.yml:4]: i dont think it matters

[the remark this answers is not in the document]
Comment (samricotta) [reply .github/workflows/copy-md.yml:4]: Okay ill change, i need to also add in 0.50 docs into this PR (or put it into another) as npm will not run with the 0.50 as an addition in the versions file

[the remark this answers is not in the document]
Comment (tac0turtle) [reply .github/workflows/copy-md.yml:4]: lets merge then do it separately. Also when we add new versions will we need to copy it over each time?

**Pull request merged**